bindings/rascal/neighbourlist/structure_manager.py

```diff
@@ -114,6 +114,31 @@ def get_features(self, calculator, species=None):
 
         return X
 
+    def get_features_gradient(self, calculator, species=None):
+        """
+        Parameters
+        -------
+        calculator : Calculator (an object owning a _representation object)
+
+        species :  list of atomic number to use for building the dense feature
+        matrix computed with calculators of name Spherical*
+
+        Returns
+        -------
+        dX_dr : ndarray of size (3*(n_neighbor+n_atom), n_features)
+            returns the gradient of representation with respect to the atomic positions that have been computed with the calculator as a dense matrix.
+        """
```

Comment:
This documentation should refer to `get_ij` as the way to get atomic indexes corresponding to the entries in the matrix returned by this function.